### ISSUE TITLE
Current price for ebs instance by region

### ISSUE THREAD

**Issue by delijati** [samuelreh/spotr#17]
Currently i use `https://github.com/arithmetric/awespottr.git`

**Comment by delijati** [comment]
Here a snippet i build https://stackoverflow.com/a/47497182/1154966

**Comment by samuelreh** [comment]
Yeah, the only issue is you can't use an image from a different region, right? One of the main benefits of using `spotr` is that it will persist the state of the machine to a launchable instance.

**Comment by delijati** [comment]
My current workflow loks like this:

1.) Finde the cheapest region
2.) Find a ubuntu image for that region
3.) Connect to it and setup the box and do some ML
4.) Destroy the box 

This has the advantage that i have no costs for persistent esb image. The disadvantage is i have to setup everything from scratch but i have a script that does that ;)
